alter_register.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE register ADD COLUMN is_link BOOLEAN NULL CHECK (is_link IN (0,1));
```

Comment:
Create `sql` directory in the root directory and move all .sql files. Prepend all files with `yyyymmdd_` (this one will be `20190811_alter_register.sql`). We will be able to rebuild the database by running all sql files in order.

Reply:
It's done on new_features branch